dev-client/app.config.ts

```diff
@@ -86,7 +86,10 @@ export default ({config}: ConfigContext): ExpoConfig => ({
   android: {
     package: 'org.terraso.landpks',
     versionCode: appVersion,
-    icon: 'src/assets/landpks-round-1024.png',
+    icon: 'src/assets/landpks-ios-1024.png',
```

Comment:
question: Is this correct? It looks like you're linking the ios icon under an android config section

Reply:
Will rename for clarity.